lib/sentimental.rb

```diff
@@ -1,23 +1,23 @@
+require 'json'
+
 class Sentimental
   attr_accessor :threshold, :word_scores, :neutral_regexps, :ngrams
 
   def initialize(threshold: 0, word_scores: nil, neutral_regexps: [], ngrams: 1)
-    if ngrams >= 1
-      @ngrams = ngrams.to_i
-    else
-      @ngrams = 1
-    end
+    @ngrams = ngrams.to_i if ngrams >= 1
+    @ngrams = 1 unless ngrams >= 1
```

Comment:
You are right, I was blinded because I don't like if else with just one line inside (neither does rubocop), other alternatives could be:
    @ngrams = 1 (outside the initialize method) 

    # inside initialize 
    @ngrams = ngrams if ngrams > 1
Or this one (I think is uglier) 
    @ngrams = (ngrams > 1) ? ngrams : 1